docs/tutorials/wiki2/distributing.rst

```diff
@@ -19,22 +19,22 @@ On Windows:
 
 .. code-block:: doscon
 
-   c:\pyramidtut> %VENV%\Scripts\python setup.py sdist
+   c:\tutorial> %VENV%\Scripts\python setup.py sdist
 
 The output of such a command will be something like:
 
 .. code-block:: text
 
    running sdist
-   # .. more output ..
+   # more output
    creating dist
    Creating tar archive
    removing 'tutorial-0.0' (and everything under it)
 
 Note that this command creates a tarball in the "dist" subdirectory named
 ``tutorial-0.0.tar.gz``.  You can send this file to your friends to show them
 your cool new application.  They should be able to install it by pointing the
-``easy_install`` command directly at it. Or you can upload it to `PyPI
+``pip install .`` command directly at it. Or you can upload it to `PyPI
```

Comment:
Fixed in https://github.com/Pylons/pyramid/pull/2881/commits/1117bcd9a458e2f8664cabeef84209cf7b168e30